dotnetclr/hatch.toml

```diff
@@ -4,7 +4,7 @@
 dependencies = [
   "datadog_checks_tests_helper @ {root:uri}/../datadog_checks_tests_helper",
 ]
-e2e-env = false
+e2e-env = true
```

Comment:
nit - that's the default value with `hatch` (more info [here](https://github.com/DataDog/integrations-core/blob/d62609c117270b810f0e75c5e8c8516fa5d78b6b/datadog_checks_dev/datadog_checks/dev/plugin/hatch/environment_collector.py)) so you can also remove this line

Reply:
Thanks for the tip! I'll go ahead and remove that line.